README.md

```diff
@@ -72,12 +72,9 @@ ng generate component my-new-component
 ng g component my-new-component # using the alias
 
 # components support relative path generation
-# if in the directory src/app/feature/ and you run
```

Comment:
Anyways since it seems to be a possible bug to an intended feature I've changed the PR to add the example instead of replacing it.